From 181617387841b695ee77b162babf9fb177002fcb Mon Sep 17 00:00:00 2001
From: toonn <toonn@toonn.io>
Date: Mon, 20 Sep 2021 11:39:46 +0200
Subject: [PATCH] test-images: Use normalization equivalent character
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

One of the test images used a combining character `ä` that can be
encoded multiple ways. This means the file's name can end up encoded
differently depending on whether/which normal form the filesystem uses.

For Nix this causes a different hash for a FOD depending on the
filesystem where it is evaluated. This is problematic because hashes
fail to match up when evaluating the FOD across multiple platforms.
---
 tests/roots/test-images/index.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/roots/test-images/index.rst b/tests/roots/test-images/index.rst
index 14a2987..219842e 100644
--- a/tests/roots/test-images/index.rst
+++ b/tests/roots/test-images/index.rst
@@ -13,7 +13,7 @@ test-image
 
    The caption of img
 
-.. image:: testimäge.png
+.. image:: testimæge.png
 
 .. image:: rimg.png
    :target: https://www.sphinx-doc.org/
-- 
2.17.2 (Apple Git-113)

